CONTRIBUTING.md

```diff
@@ -287,9 +287,12 @@ Any follow-up changes to experimental components will not be considered breaking
 <details>
 <summary>When should documentation be created and published?</summary>
 
-Each Bpk component has a corresponding README file which contains information about the component such as usage examples and API documentation. Our components' full documentation is at [skyscanner.design](https://www.skyscanner.design). New experimental components should have a README file, but don’t need to be published to [skyscanner.design](https://www.skyscanner.design). Instead, when an experiment has run and is considered successful and so the change is stable, documentation can be published.
+Each Bpk component has a corresponding README file which contains information about the component such as usage examples and API documentation. Our components' full documentation is at [skyscanner.design](https://www.skyscanner.design). New experimental components should have a README file, but don’t need to be published to [skyscanner.design](https://www.skyscanner.design). Make sure the README file reflects the component is experimental! When an experiment has run and is considered successful and so the change is stable, documentation can be published.
 
 For changes to existing components, make sure the API documentation is updated to indicate if something is experimental.
+    
+Major changes will often require a migration guide. If an experiment is considered succesful, you should add a migration guide within the docs folder located in the respective component folder.
```

Comment:
Maybe:
Major changes should be done via deprecation if possible and will often require a migration guide.

Reply:
It actually does say above `For major changes, you should create a new experimental V2 component. If the experiment is successful, the old component should be deprecated.`
This paragraph is mainly about docs